defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 
 interfaces_pkgs:
   debian:
-    - python-selinux
+    - python{% if ansible_python_version >= '3' %}3{% endif %}-selinux
```

Comment:
Better to do a numeric comparison or full version comparison:

`ansible_python.version.major >= 3`